### PR TITLE
Fixes #4513 : Dark mode implementation - Dialogs, Status Bar

### LINKED ISSUE #4513: Add Dark Mode support for BottomSheetFragment (Exploration + Revision Card)
Add Dark Mode support for BottomSheetFragment in both Exploration and Revision Card. 

### Current UI

https://user-images.githubusercontent.com/43074241/185469039-24e440da-69f9-4885-938c-688e3ff3ecf6.mp4

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fixes #4513 : Dark mode implementation - Dialogs, Status Bar

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

#### Hint and Solutions Dialog

   

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@rt4914 Can you PTAL on this PR.

File: app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml [hunk 1]
```diff
@@ -7,7 +7,8 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:orientation="vertical"
-    android:padding="8dp">
+    android:padding="8dp"
+    android:background="@color/color_palette_primary_container_background_color">
```
Comment (rt4914) [inline app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml]: I think this is by mistake. Use component color here.

File: app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml [hunk 2]
```diff
@@ -17,9 +18,10 @@
       android:drawablePadding="8dp"
       android:gravity="center_vertical"
       android:text="@string/menu_options"
-      android:textColor="@color/color_def_black"
+      android:textColor="@color/component_color_shared_primary_dark_text_color"
       android:textSize="20sp"
-      app:drawableStartCompat="@drawable/ic_options_icon_grey_24dp" />
+      app:drawableStartCompat="@drawable/ic_options_icon_grey_24dp"
+      android:drawableTint="@color/component_color_shared_multipane_icon_color" />
```
Comment (rt4914) [inline app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml]: Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

**Review by rt4914 — APPROVED** [review]
LGTM, Suggested minor fixes.

File: app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml (hunk 1 above)
Comment (MohitGupta121) [reply app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml]: Done.

File: app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml (hunk 2 above)
Comment (MohitGupta121) [reply app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml]: Done.

**Comment by MohitGupta121** [comment]
@rt4914 Done the review changes. PTAL.
Thanks.

**Pull request merged**